packages/esm-patient-common-lib/src/siderail-nav-button/siderail-nav-button.component.tsx

```diff
@@ -56,13 +69,13 @@ export const SiderailNavButton: React.FC<SiderailNavButtonProps> = ({
       className={classNames(styles.container, {
         [styles.active]: isWorkspaceActive,
       })}
+      enterDelayMs={300}
       kind="ghost"
       label={label}
       onClick={handler}
     >
       <div className={styles.elementContainer}>
-        {getIcon({ size: 20 })}
-        <span className={styles.countTag}>{formOpenInTheBackground ? '!' : tagContent}</span>
+        <Tags isTablet={false} />
```

Comment:
@vasharma05 There' s no real superiority in calling `{Tags()}` rather than `<Tags />`. The accepted answer on that SO post is... not wrong, but terrible advice (and, importantly, in that one case, the component being rendered like that always produces the same mark-up) .

The right way to approach this is to extract `Tags` into it's own component (it can be defined in the same file) and pass the data it needs into the component as props. Then we're just using React as designed.